tile_generator/config_unittest.py

```diff
@@ -149,6 +151,13 @@ def new_comparer(expected, given, path):
 				self.assertEquals(expected, given, (path, 'Expected to have the value:\n%s\nHowever, instead got:\n%s' % (expected, given)))
 
 
+                # Confirm that the new consumes: contains 'redis'
+		for release in cfg['releases']:
+                    if release.has_key('consumes_for_deployment'):
+                        self.assertIn("'from': 'redis'", str(release['consumes_for_deployment']['redis']))
```

Comment:
Now that `'optional': False` is removed the `assertIn` should be changed to a `self.assertDictEqual({'from': 'redis'}, release['consumes_for_deployment']['redis'])`

Reply:
Fixed..